DDDEastAnglia/Views/Home/Sponsors.cshtml

```diff
@@ -1,68 +1,20 @@
-﻿@{
+﻿@using DDDEastAnglia.Controllers
+@model IEnumerable<DDDEastAnglia.Models.SponsorModel>
+
+@{
     ViewBag.Title = "Our Sponsors";
 }
 
 <h2>@ViewBag.Title</h2>
 <p>We would like to extend our thanks to our generous sponsors without whose support @Html.DDDEastAnglia() would not be possible.</p>
 
-<p>If you would like to sponsor @Html.DDDEastAnglia(), please @Html.ActionLink("get in touch", "Contact").</p>
-
-<div class="sponsor span5">
-    <a href="http://careers.comparethemarket.com"><img src="@Url.Content("~/Images/sponsors/comparethemarket.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://redg.at/11xaIPC"><img src="@Url.Content("~/Images/sponsors/redgate.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.ndc-london.com/"><img src="@Url.Content("~/Images/sponsors/ndclondon.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.techsmith.com"><img src="@Url.Content("~/Images/sponsors/techsmith.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.tinamous.com"><img src="@Url.Content("~/Images/sponsors/tinamous.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.credit360.com/dddeastanglia"><img src="@Url.Content("~/Images/sponsors/CRedit360.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.equalexperts.com"><img src="@Url.Content("~/Images/sponsors/equalexperts.png")" /></a>
-</div>
+<p>If you would like to sponsor @Html.DDDEastAnglia(), please @Html.ActionLink("get in touch", "Contact", "Home").</p>
 
-<div class="sponsor span5">
-    <a href="http://www.huddle.com"><img src="@Url.Content("~/Images/sponsors/huddle.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.blackdotsolutions.co.uk"><img src="@Url.Content("~/Images/sponsors/blackdotsolutions.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.github.com"><img src="@Url.Content("~/Images/sponsors/github.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.devexpress.com"><img src="@Url.Content("~/Images/sponsors/devexpress.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="https://zud.io"><img src="@Url.Content("~/Images/sponsors/zudio.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.pluralsight.com"><img src="@Url.Content("~/Images/sponsors/pluralsight.png")" /></a>
-</div>
-
-<div class="sponsor span5">
-    <a href="http://www.skillsmatter.com"><img src="@Url.Content("~/Images/sponsors/skillsmatter.png")" /></a>
-</div>
+@foreach (var sponsor in Model)
+{<div class="sponsor span5">
```

Comment:
No, just a CRLF to put the `div` tag on the next line (plus indented correctly).